### PR TITLE
v3.17.0

### PR DESCRIPTION
**Pull request by jsanford8**
- prepare for upcoming feature release
 - update docs around default measurement

### CODE REVIEW

File: .openapi-generator/FILES [hunk 1]
```diff
@@ -373,10 +381,10 @@ lib/mux_ruby/models/web_input.rb
...
 lib/mux_ruby/version.rb
 mux_ruby.gemspec
+spec/api/drm_configurations_api_spec.rb
 spec/api_client_spec.rb
 spec/configuration_spec.rb
-spec/models/update_live_stream_new_asset_settings_spec.rb
-spec/models/update_user_agent_restriction_request_spec.rb
-spec/models/user_agent_restriction_request_spec.rb
-spec/models/user_agent_restriction_settings_spec.rb
+spec/models/drm_configuration_response_spec.rb
```
Comment (jsanford8) [inline .openapi-generator/FILES:387]: @philcluff this doesn't seem right... not sure why update live stream and user agent restrictions... this makes me think we missed a cherry-pick

Comment (jsanford8) [reply .openapi-generator/FILES:387]: the files still exist...

Comment (jsanford8) [reply .openapi-generator/FILES:387]: yeah I'm at a loss here, I don't know what's happening

Comment (philcluff) [reply .openapi-generator/FILES:387]: This `FILES` list seems to get out of sync fairly often, I'm unclear if it's actually used for anything.

**Review by philcluff — APPROVED** [review]
LGTM.

Should we drop Ruby 2.7 support?

**Comment by jsanford8** [comment]
Yup, that's my plan now, going to do that now

**Pull request merged**